runtime/parser/expression.go

```diff
@@ -815,15 +815,18 @@ func defineIdentifierExpression() {
 				), nil
 
 			case keywordView:
```

Comment:
Nice! Needs to be "duplicated" in `parseStatement`

Reply:
ah damn you're right, this bit me last time too